docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -445,7 +445,28 @@ The top-level properties are:
 		"inputObjects": "camelCase"
 	},
 	"pruneGeneratedFiles": true,
-	"markOperationDefinitionsAsFinal": true
+	"markOperationDefinitionsAsFinal": true,
+	"schemaCustomization" : {
+    "customTypeNames" : {
```

Comment:
Not sure what's going on with the indentation here and below, double checked it locally and it looks fine.